src/types/das-types.ts

```diff
@@ -145,7 +145,7 @@ export interface SearchAssetsRequest {
     items: FullRwaAccount
   };
   export type GetAssetResponseList = {
-    grand_total?: boolean;
+    grand_total?: number | undefined;
```

Comment:
As `grand_total` is already nullable there is no need to specify `undefined`

Reply:
Ah, that's my bad — resolved now!

Reply:
LGTM!